BabelfishFeatures.cfg

```diff
@@ -1207,6 +1209,8 @@ report_group=DML
 rule=top_clause
 list=0,100,NONZERO,VARIABLE,EXPRESSION
 supported-1.0.0=100
+supported-4.8.0=*

```

Comment:
this should be `supported-4.8.0-4.*=`

it makes no sense to have `supported-4.8.0=*` and `supported-5.4.0=*`  -- the latter is then effectively redundant. But it would also mean that 5.1/5.2/5.3 support this feature as well, and that is not the case.